gin/sentrygin.go

```diff
@@ -61,7 +60,7 @@ func (h *handler) handle(c *gin.Context) {
 	}
 
 	transaction := sentry.StartTransaction(ctx,
-		fmt.Sprintf("%s %s", c.Request.Method, c.Request.URL.Path),
+		c.FullPath(),
```

Comment:
Any specific reason you removed the HTTP method verb?

Reply:
Yes, cause it's unnecessary for HTTP Method in transaction name. 
There is two reasons for that: 
1) HTTP Method is already present in translation and no need to duplicate this info 
2) According to [screenshots from sentry docs](https://docs.sentry.io/product/performance/transaction-summary/?original_referrer=https%3A%2F%2Fwww.google.com%2F) transaction name should consist of endpoint without HTTP Method

Reply:
I see, I would still keep it, because:

- Not all places in the UI show the `HTTP method`; Hence it is hard to distinguish between transactions with different HTTP methods.
- This is a breaking change, as it will change the behaviour of a configured `traces_sampler` as well as dashboards or alerts created in the product. For the sake of better bucketing, I'm ok doing this, but I wanted to inform you of the implications.

Reply:
What do you think about giving developers the ability to turn on/off those transactions?
This way I can implement transaction as I want it, but to do that I need the way of disabling built-in transaction.

Reply:
I think I can do it in separate PR

Reply:
We have certain expectations in the product for how a transaction should be structured. In the case of Go, this is a big problem, as seen in your screenshot above. The `/*` is applied during ingestion, where we perform server-side bucketing to group similar transactions together. This is expensive.
Hence we absolutely do not want to make this process a bigger foot gun as it already is 🙂 

For now, please keep the HTTP verbs. The change is already a massive improvement from before.

Reply:
Ok, no problem, I can live with HTTP method. Will return it

Reply:
Done